layouts/partials/sections/education-alt.html

```diff
@@ -106,6 +106,13 @@ <h6 class="text-heading">{{ i18n "extracurricular_activities" }}:</h6>
                                 </ul>
                             </div>
                             {{ end }}
+                            {{ if .custonSections }}
+                            <div class="extracurricular-activities">
+                                {{ range .custonSections }}
+                                    <h6><span class="text-heading">{{ .name }}: </span><span>{{ .content }}</span></h6>
```

Comment:
Let's add a separate div here with heading and content. The content can be markdown so that user can add list,table etc. This will also make it consistent with other sections.

For example:
```html
<div class="custom-section">
  <h6 class="text-heading">{{.name }}:</h6>
  <div>
   {{ .content | markdownify }}
  </div>
</div>
```